vic/drivers/image/src/vic_init_output.c

```diff
@@ -193,10 +207,167 @@ initialize_history_file(nc_file_struct *nc)
         log_err("Error defining time dimenension in %s", nc->fname);
     }
 
+    // define the netcdf variable time
+    status = nc_def_var(nc->nc_id, "time", NC_INT, 1,
+                        &(nc->time_dimid), &(time_var_id));
+    if (status != NC_NOERR) {
+        log_err("Error defining time variable in %s", nc->fname);
+    }
+    status = nc_put_att_text(nc->nc_id, time_var_id, "standard_name",
+                             strlen("time"), "time");
+    if (status != NC_NOERR) {
+        log_err("Error adding attribute in %s", nc->fname);
+    }
+    if (global_param.output_steps_per_day == 1) {
+        sprintf(strUnit, "days");
+    }
+    else {
+        sprintf(strUnit, "hours");
+    }
+    sprintf(str, "%s since %i-%02i-%02i 00:00:00", strUnit,
+            global_param.startyear, global_param.startmonth,
+            global_param.startday);
```

Comment:
This date doesn't need to be intuitive or even seem like a real date.  It only needs to be a valid date in the calendar (which "0001-01-01" for all calendars). Most users will not interact with this date string at all, it is most frequently used by software that interprets the time variable.  Choosing "0001-01-01" has two main advantages:

1) It is the earliest valid date for all calendars
2) It allows VIC to be run before 1900 without producing negative timestamps (I've seen problems before)

"0001-01-01" is pretty commonly used in the climate modeling community.